mobile/library/cc/engine_builder.cc

```diff
@@ -221,7 +221,7 @@ EngineBuilder& EngineBuilder::enforceTrustChainVerification(bool trust_chain_ver
   enforce_trust_chain_verification_ = trust_chain_verification_on;
   return *this;
 }
-
+#ifdef ENVOY_GOOGLE_GRPC
```

Comment:
Not a blocker, and probably out of scope for this PR, but ideally we'd have a dedicated `ENVOY_MOBILE_XDS` define / build flag that would force `ENVOY_GOOGLE_GRPC` to be enabled and allow decoupling EM's xDS support from google_grpc.